packages/myst-cli/src/transforms/links.ts

```diff
@@ -154,7 +154,8 @@ export class StaticFileTransformer implements LinkTransformer {
     const [linkFile, ...target] = linkFileWithTarget.split('#');
     const { url, title, dataUrl } =
       selectors.selectFileInfo(this.session.store.getState(), linkFile) || {};
-    if (url != null) {
+    // If the link is non-static, and can be resolved locally
+    if (url != null && !url.static) {
```

Comment:
URL here is a string? Maybe do you mean `link.static`?

```suggestion
    if (url != null && link.static !== true) {
```

Reply:
Yes ... how ... ?!

I've got a few PRs in flight, but I never recall writing `url.static` ha!